src/StackExchange.Redis/ServerEndPoint.cs

```diff
@@ -556,7 +556,27 @@ internal bool CheckInfoReplication()
         }
 
         private int lastInfoReplicationCheckTicks;
-        internal volatile bool ForceReplicationCheck;
+        internal volatile int ConfigCheckSeconds;
+        [ThreadStatic]
+        private static Random r;
+
+
+        // Forces frequent replication check starting from 1 second upto max ConfigCheckSeconds with an exponential increment
+        internal void ForceExponentiallyReplicationCheck()
```

Comment:
Can we rename this for clarity please?
```suggestion
        internal void ForceExponentialBackoffReplicationCheck()
```